Packs/SumoLogic_Cloud_SIEM/Classifiers/classifier-Sumo_Logic_Cloud_SIEM.json

```
@@ -1,34 +1,17 @@
 {
 	"defaultIncidentType": "",
 	"definitionId": "",
-	"description": "Classifier for Sumo Logic Insight",
+	"description": "Classifier for Sumo Logic Insight and Signal",
 	"feed": false,
 	"fromVersion": "6.2.0",
-	"id": "Sumo Logic Insight Classifier",
+	"id": "Sumo Logic Cloud SIEM Classifier",
```

As discussed on the demo, please revert the name/ID from the classifier.

```suggestion
	"id": "Sumo Logic Insight Classifier",
```

---

Reverted